src/main/kotlin/view/InputView.kt

```diff
@@ -2,21 +2,40 @@ package view
 
 object InputView {
 
+    private const val MANUAL_LOTTO_NUMBERS_REQUEST_MESSAGE = "수동으로 구매할 번호를 입력해 주세요."
+    private const val MANUAL_LOTTO_COUNT_REQUEST_MESSAGE = "수동으로 구매할 로또 수를 입력해 주세요."
     private const val PURCHASE_MONEY_REQUEST_MESSAGE = "구입금액을 입력해 주세요."
     private const val CATCH_NUMBER_REQUEST_MESSAGE = "지난 주 당첨 번호를 입력해 주세요."
     private const val BONUS_NUMBER_REQUEST_MESSAGE = "보너스 볼을 입력해 주세요."
     private const val TOKENIZER = ","
+    private const val INITIAL_PURCHASING = 0
+
+    private fun requestNumbers(): List<String> = readln().split(TOKENIZER).map { it.trim() }
```

Comment:
```suggestion
    private fun requestNumbers(): List<String> = readln().split(TOKENIZER).map(String::trim)
```
메서드 참조를 활용하시면 위와 같이 사용할 수 있어요.